class_defines.py

```diff
@@ -2247,43 +2259,26 @@ def _get_angle(A, B):
 
         return math.degrees(math.acos(x))
 
-    @staticmethod
-    def _get_angle_inv(A, B):
-        # (A dot B)/(|A||B|) = -cos(valA)
-        divisor = A.length * B.length
-        if not divisor:
-            return 0.0
-
-        x = -A.dot(B) / divisor
-        x = max(-1, min(x, 1))
-
-        return math.degrees(math.acos(x))
-
     def init_props(self):
         # Set initial angle value to the current angle
         line1, line2 = self.entity1, self.entity2
-
         vec1, vec2 = line1.direction_vec(), line2.direction_vec()
         angle_std = self._get_angle(vec1, vec2)
-        angle_inv = self._get_angle_inv(vec1, vec2)
-
-        setting = angle_inv < angle_std
-        angle = angle_inv if setting else angle_std
-
-        self.value = angle
```

Comment:
the Operator will soon use the return value to update this, so we don't need it here